plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -357,16 +357,10 @@
     "divideInteger": {
         "cpu": {
             "arguments": {
-                "constant": 196500,
-                "model": {
-                    "arguments": {
-                        "intercept": 453240,
-                        "slope": 220
-                    },
-                    "type": "multiplied_sizes"
-                }
+                "intercept": 453240,
+                "slope": 220
             },
-            "type": "const_above_diagonal"
+            "type": "multiplied_sizes"
```

Comment:
Does changing the JSON file like this not break everything? I think we need to record both the old shape (for use by PlutusV1 and V2 before PV9) and the new shape?

Reply:
Oh, there's something wrong there. I'll check that later.

Reply:
> `builtinCostModel.json`, one for each `PlutusV<n>`

Not just `PlutusV<n>`, though? We also need to worry about protocol versions. And I don't think we need to worry about all of `PlutusV<n>`. I think the end result is that for Conway we'll need 3 files (it's a coincidence that we have exactly that many Plutus versions). I'll write about that later.

Reply:
We need one JSON per semantic variant, so 3 sounds right. Of course, we can merge them into a single JSON file, but all the information needs to be there.